tracing/tracing.go

```diff
@@ -129,7 +113,8 @@ func HeaderMatcher(key string) (string, bool) {
 func trimPodName(p string) string {
 	a := strings.Split(p, "-")
 	i := len(a)
-	if i > 2 {
+
+	if len(a[(i-1)]) == 5 && (len(a[(i-2)]) == 10 || len(a[(i-2)]) == 11) {
```

Comment:
Please add a comment here and/or use constants to explain the magic numbers

Reply:
comment added